gensim/summarization/bm25.py

```diff
@@ -18,7 +18,8 @@ class BM25(object):
 
     def __init__(self, corpus):
         self.corpus_size = len(corpus)
-        self.avgdl = sum(map(lambda x: float(len(x)), corpus)) / self.corpus_size
+        self.avgdl = sum(map(lambda x: float(len(x)), corpus)
```

Comment:
PEP8 error E501:Line longer than 79 characters.

Does this line need modification?